lib/xml/utils/node_type.dart

```diff
@@ -2,13 +2,62 @@ library xml.utils.node_type;
 
 /// Enum of the different XML Node types.
 enum XmlNodeType {
+  /// An element start or self-closed tag, e.g. `<item>` or `<item />`.
   ELEMENT,
+
+  /// An element closing tag, e.g. `</item>`.
+  END_ELEMENT,
```

Comment:
Is this still needed?

Reply:
Yes, it signals that the end of an element is reached.